src/main/kotlin/com/craftschemers/hub/commands/JoinCommand.kt

```diff
@@ -0,0 +1,37 @@
+package com.craftschemers.hub.commands
+
+import com.craftschemers.hub.minigame.Minigame
+import org.bukkit.ChatColor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+
+class JoinCommand(
+    override val name: String = "join",
+    override val aliases: Array<String>? = null,
+    override val canBeConsole: Boolean = false,
+    override val description: String = "Use this command to join a minigame",
+    override val parameters: Array<String>? = null,
+    override val usage: Array<String> = arrayOf("/hub join <Minigame>"),
+    override val permissionMessage: String = "Permission message",
+    override val permission: String = "permission"
+) : ICommand {
+
```

Comment:
Do we want to be able to instantiate multiple `JoinCommand`s with different values? If not this whole thing could just be an `object` like this:

```kotlin
object JoinCommand : ICommand {
    override val name = "join"
    override val aliases: Array<String>? = null
    override val canBeConsole = false
    // ...
```

Reply:
not sure im following.. why would we want multiple `JoinCommand`s?